From 0989f13ecbc2155b05da0cb091801d32d750d2d4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rapha=C3=ABl=20M=C3=A9lotte?=
 <raphael.melotte@essensium.com>
Date: Thu, 29 Aug 2019 09:51:02 +0200
Subject: [PATCH] Makefile: add an install_libs target.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This allows to install only the library files (if you need pymupdf but
not mupdf binaries for example).

Signed-off-by: Raphaël Mélotte <raphael.melotte@essensium.com>
---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c5bc541ac..e14f88526 100644
--- a/Makefile
+++ b/Makefile
@@ -322,7 +322,7 @@ libs: $(INSTALL_LIBS)
 tools: $(TOOL_APPS)
 apps: $(TOOL_APPS) $(VIEW_APPS)
 
-install: libs apps
+install_libs:
 	install -d $(DESTDIR)$(incdir)/mupdf
 	install -d $(DESTDIR)$(incdir)/mupdf/fitz
 	install -d $(DESTDIR)$(incdir)/mupdf/pdf
@@ -333,6 +333,7 @@ install: libs apps
 	install -d $(DESTDIR)$(libdir)
 	install -m 644 $(INSTALL_LIBS) $(DESTDIR)$(libdir)
 
+install: libs apps install_libs
 	install -d $(DESTDIR)$(bindir)
 	install -m 755 $(TOOL_APPS) $(VIEW_APPS) $(DESTDIR)$(bindir)
 
-- 
2.21.0

